projects/core/src/auth/user-auth/user-auth.module.ts

```diff
@@ -6,6 +6,7 @@ import { provideDefaultConfig } from '../../config/config-providers';
 import { provideConfigValidator } from '../../config/config-validator/config-validator';
 import { baseUrlConfigValidator } from './config/base-url-config-validator';
 import { defaultAuthConfig } from './config/default-auth-config';
+import { UserAuthEventModule } from './events/index';
```

Comment:
Maybe it's better to import directly from the file, as Ivy is stricter about this than the View engine.